questions/qa-personal-names.fr.html

```diff
@@ -100,8 +100,7 @@ <h3 id="partorder"><a href="#partorder">Variation de l'ordre des éléments</a><
   <section>
     <h3 id="multiplefamily"><a href="#multiplefamily">Noms de famille multiples</a></h3>
     <p>Les hispanophones ont couramment deux noms de famille. Par exemple, <b class="highlight">María José Carreño Quiñones</b> peut être la fille d'Antonio Carreño Rodríguez et de María Quiñones Marqués.</p>
-    <!--p>Vous la nommerez Señorita Carreño, et non Señorita Quiñones.</p-->
-	<p>You could refer to her as Señorita Carreño, but not Señorita Quiñones. However, more recently there is also a preference to move away from titles that indicate the married status of women (especially when they are not so indicated for men), so 'Señorita' may be replaced with 'Señora'.</p>
+    <p>Vous pourriez la nommer Señorita Carreño mais pas Señorita Quiñones. Cependant, la préférence pour une distanciation avec les titres dévoilant le statut marital des femmes est apparu récemment (surtout dans la mesure où ils ne sont pas distingués pour les hommes). «&nbsp;Señorita&nbsp;» pourra donc être remplacé par «&nbsp;Señora&nbsp;».</p>
```

Comment:
Warning: in French punctuation must be WITHIN the quotes.
```suggestion
    <p>Vous pourriez la nommer Señorita Carreño mais pas Señorita Quiñones. Cependant, la préférence pour une distanciation avec les titres dévoilant le statut marital des femmes est apparu récemment (surtout dans la mesure où ils ne sont pas distingués pour les hommes). «&nbsp;Señorita&nbsp;» pourra donc être remplacé par «&nbsp;Señora.&nbsp;»</p>
```

Reply:
Roger that.

Reply:
In fact, [only entire sentences][1] would have their period included before ` »` ; same goes for parenthesis.

In this case and in general, unless it is a quote, I would be happy to use italic instead of `« ` and ` »`.

In this case, the text would become :
```
<i>Señorita</i> pourra donc être remplacé par <i>Señora</i>.</p>
```

[1]: https://fr.wikipedia.org/wiki/Wikip%C3%A9dia:Conventions_typographiques#Points

Reply:
I'll ask someone in the know.
Ping @emmanuelc ?

Reply:
I asked @emmanuelclement and it seems I was saying nonsense. Don't mind me. ;)